src/OSPSuite.Core/Domain/Model.cs

```diff
@@ -9,6 +9,16 @@ public interface IModel : IObjectBase, IMolWeightFinder
    {
       IContainer Root { get; set; }
       IContainer Neighborhoods { set; get; }
+
+      /// <summary>
+      ///    Returns the Body weight <see cref="IParameter" /> if available in the model otherwise null.
+      /// </summary>
+      IParameter BodyWeight { get; }
```

Comment:
There are the two properties that are required. BodyWeight PArmaeter and TotdalDrugMass